app/assets/stylesheets/application.scss

```diff
@@ -1,95 +1,91 @@
-// govuk_frontend_toolkit helpers
-@import "typography";
-@import "measurements";
-@import "colours";
-@import "shims";
-@import "css3";
-@import "grid_layout";
-
-@import "reset";
+$govuk-compatibility-govuktemplate: true;
 @import 'govuk_publishing_components/all_components';
 
 // some colours used below
-$manual-search-button-colour: #0b0c0c;
 $manual-search-button-border-colour: #222;
 
 %common-button-styles {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   font-size: 100%;
   border-width: 0;
-  color: $govuk-blue;
+  color: $govuk-link-colour;
   background: none;
   cursor: pointer;
+
+  &:focus {
+    @include govuk-focused-text;
+  }
 }
 
 main {
   display: block;
 }
 
 .manuals-frontend-body {
-  padding-bottom: $gutter;
+  padding-bottom: govuk-spacing(6);
 
   header {
-    background: $govuk-blue;
+    background: $govuk-brand-colour;
 
     &.hmrc {
-      background: $hm-revenue-customs;
+      background: govuk-organisation-colour("hm-revenue-customs", $websafe: false);
     }
 
-    padding: $gutter;
-    color: $white;
+    padding: govuk-spacing(6) govuk-spacing(3);
```

Comment:
This is a different padding from the original. Is that deliberate?

Edit: oh, I think I understand, this is to do with the change below to the media queries?

Reply:
yes, it's to do with the MQ change